framework/src/main/java/org/checkerframework/framework/ajava/AjavaUtils.java

```diff
@@ -0,0 +1,50 @@
+package org.checkerframework.framework.ajava;
+
+import com.github.javaparser.ast.CompilationUnit;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.body.EnumDeclaration;
+import com.github.javaparser.ast.body.TypeDeclaration;
+import com.sun.tools.javac.code.Symbol.ClassSymbol;
+import java.util.Optional;
+import javax.lang.model.element.TypeElement;
+import org.checkerframework.javacutil.BugInCF;
+
+/** A utility class for working with javac and JavaParser representations of a source file. */
+public class AjavaUtils {
+    /**
+     * Returns the path to the source file containing {@code element}
+     *
+     * @param element the type element to look at
+     * @return path to the source file containing {@code element}
+     */
+    public static String getSourceFilePath(TypeElement element) {
```

Comment:
This method does not seem specific to JavaParser.  I would put it in ElementUtils.